### ISSUE TITLE
Fabric / Answers confusion

### ISSUE THREAD

**Issue by garnett** [orta/ARAnalytics#221]
Looks like "Answers" is more about analytics + Crashlytics has a bunch of methods deprecated:
```- (void)logEvent:(NSString *)eventName CLS_DEPRECATED("Please refer to Answers +logCustomEventWithName:");
```

Should I modify existing `Fabric` provider in favour of these changes or I need new `Answers` provider?

**Comment by orta** [comment]
Sure for either, I don't know too much about the Fabric integration, whatever you feel is best :+1:

**Comment by garnett** [comment]
@orta I see :smile: Even though ```Fabric``` is not the best name for this particular provider, nobody (me) wants to mess with renaming it to ```Crashlytics``` and migrating to new version etc.

However, ```Answers``` is completely unrelated to this and should be separate provider, will do PR

**Comment by cemaleker** [comment]
What is the update on this. I might improve Fabric / Crashlytics implementations if needed.

**Comment by garnett** [comment]
In my project, I've made separate `Crashlytics` provider with purpose of adding logs to crash reports.
Is that something you find useful @cemaleker?

**Comment by cemaleker** [comment]
@garnett Yes definately. But I also want to track events with Answers. I think I can do this with the current Answers provider, right?

**Comment by garnett** [comment]
@cemaleker that's correct, even thought it is called `FabricProvider` it triggers Answers methods inside.
